### PR TITLE
Documentation for PDB support for custom resource with scale subresource

### PR DESCRIPTION
**Pull request by mortent**
KEP: https://github.com/kubernetes/enhancements/blob/master/keps/sig-apps/20190412-pdb-support-for-custom-resources-with-scale-subresource.md
Tracking issue: https://github.com/kubernetes/enhancements/issues/981
Merged PR: https://github.com/kubernetes/kubernetes/pull/76294

/milestone 1.15
cc @christianh814

### CODE REVIEW

File: content/en/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions.md [hunk 1]
```diff
@@ -660,6 +660,8 @@ kubectl get crontabs my-new-cron-object -o jsonpath='{.spec.replicas}'
 5
 ```
 
+Custom resources with the scale subresource enabled can also be protected with [PodDisruptionBudgets](docs/tasks/run-application/configure-pdb/).
```
Comment (sftim) [inline content/en/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions.md]:
```suggestion
You can use a [PodDisruptionBudget](docs/tasks/run-application/configure-pdb/) to protect custom resources that have the scale subresource enabled.
```

**Comment by makoscafee** [comment]
Hi @mortent , @Bradamant3, @sftim  can this be approved at the state it is right now? We are looking to have all PR for release 1.15 merged by June 10th. Let me know if there is anything docs team can assist.

**Comment by sftim** [comment]
Readable English, valid Markdown, it makes sense to me.
/lgtm

**Pull request merged**